### PR TITLE
Redone pleasebackup view following the mockup

### PR DESCRIPTION
**Pull request by ncoelho**
Follows the floating layout, avoiding overriding words when these are bigger than the column space, add badges to the background to facilitate reading

### CODE REVIEW

**Comment by Overtorment** [comment]
nobody says `mnemonic sentence`
where did u get this bs

**Comment by ncoelho** [comment]
> nobody says `mnemonic sentence`
> where did u get this bs

Trying to put it in more understandable words. A "mnemonic" is a tool to remember something, can be a song, a trick, a device, a visual... In this case is a sentence.

not seeing the bs there :D

**Comment by Overtorment** [comment]
@ncoelho conflicts

**Comment by Overtorment** [comment]
conflicts

**Comment by Overtorment** [comment]
pls fix conflicts

**Comment by Overtorment** [comment]
conflict

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ https://appetize.io/app/ym2h5q5hz8yz584etme27xzxtr

(posted automatically)

**Comment by ncoelho** [comment]
this build is not ok on android

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ https://appetize.io/app/eb0hrppryap4r8rn2yw9c2yvuw

(posted automatically)

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ https://appetize.io/app/7amkbk6a4j4k5k2z95ae98prtc

(posted automatically)

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ https://appetize.io/app/0rdegh79kcjz7fj2x7q04wycpm

(posted automatically)

File: loc/cs_CZ.js [hunk 1]
```diff
@@ -226,10 +226,11 @@ module.exports = {
     sameWalletAsInvoiceError: 'You can not pay an invoice with the same wallet used to create it.',
   },
   pleasebackup: {
-    title: 'Your wallet is created...',
+    title: 'Backup',
+    success: 'Success, your wallet is now created!',
     text:
-      "Please take a moment to write down this mnemonic phrase on a piece of paper. It's your backup you can use to restore the wallet on other device. You can use Electrum wallet on desktop (https://electrum.org/) to restore the same wallet.",
-    ok: 'OK, I wrote this down!',
+      "Please take a moment to write down this mnemonic sentence on paper. It’s your backup that you can use to restore your wallet.",
```
Comment (Overtorment) [inline loc/cs_CZ.js]: stupid SENTENCE everyvhere instead of a PHRASE

**Comment by ncoelho** [comment]
Should be “seed” instead, or “phrase”?

**Comment by Overtorment** [comment]
Im on it

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ https://appetize.io/app/uevd48zqg28n94ux4gc99ww5bw

(posted automatically)

File: screen/wallets/pleaseBackup.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import React, { Component } from 'react';
 import { ActivityIndicator, View, BackHandler, Text } from 'react-native';
 import { BlueSpacing20, SafeBlueArea, BlueNavigationStyle, BlueText, BlueButton } from '../../BlueComponents';
+import { Badge } from 'react-native-elements';
 import PropTypes from 'prop-types';
 import Privacy from '../../Privacy';
+import { ScrollView } from 'react-native-gesture-handler';
```
Comment (marcosrdz) [inline screen/wallets/pleaseBackup.js]: Change to FlatList.

Comment (marcosrdz) [reply screen/wallets/pleaseBackup.js]: If you don't want to change to FlatList right now, then change the ScrollView import from gesture handler to react native.

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ https://appetize.io/app/wpkkzaa7f35t7xbxfdkjhb0erm

(posted automatically)

**Pull request merged**